src/ITensorChainRules/ITensorChainRules.jl

```diff
@@ -3,25 +3,185 @@
 using ChainRulesCore
 using ITensors
 
+# Needed for defining the rule for `adjoint(A::ITensor)`
+# which currently doesn't work by overloading `ChainRulesCore.rrule`
+using ZygoteRules: @adjoint
```

Comment:
I suggest overloading operations that can only be defined via adjoint in another file, so separating those from rrule

Reply:
Sure, makes sense.